packages/run-protocol/src/runStake/runStakeKit.js

```diff
@@ -293,7 +293,7 @@ const helperBehavior = {
     );
     assert(
       giveRUNonly || AmountMath.isGTE(newMaxDebt, newDebt),
-      `cannot borrow ${q(newDebt)} against ${q(amountLiened)}; max is ${q(
+      X`cannot borrow ${q(newDebt)} against ${q(amountLiened)}; max is ${q(
```

Comment:
These disclose actual amounts, which we should generally consider sensitive.

```suggestion
      X`cannot borrow ${newDebt} against ${amountLiened}; max is ${
```

Reply:
done